neuralprophet/forecaster.py

```diff
@@ -1858,22 +1873,22 @@ def plot_latest_forecast(
             fcst, self.config_train.quantiles, n_last=1 + include_previous_forecasts
         )
 
-        # Check whether the default plotting backend is overwritten
-        plotting_backend = (
-            plotting_backend
-            if plotting_backend != "default"
-            else (self.plotting_backend if hasattr(self, "plotting_backend") else "matplotlib")
-        )
+        # Check whether a local or global plotting backend is set.
+        plotting_backend = auto_set_plotting_backend(plotting_backend)
+        if hasattr(self, "plotting_backend"):
+            plotting_backend = auto_set_plotting_backend(self.plotting_backend)
```

Comment:
Why do you call `auto_set_plotting_backend` a second time here? For me this reads as the local parameter would be overridden by the global configuration (which is not what we want I assume)?

Reply:
@noxan You are absolutely right! What do you think about the following:
```
plotting_backend = auto_set_plotting_backend(plotting_backend)
if hasattr(self, "plotting_backend") and plotting_backend is None:
     plotting_backend = auto_set_plotting_backend(self.plotting_backend)
```